spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -254,10 +254,9 @@ public long executeTask(String taskName, Map<String, String> taskDeploymentPrope
 
 		// We now have a new props and args what should really get used.
 		Map<String, String> mergedTaskDeploymentProperties = report.getMergedDeploymentProperties();
-		List<String> mergedCommandLineArguments = report.getMergedCommandLineArguments();
 
 		taskExecutionInformation.setTaskDeploymentProperties(mergedTaskDeploymentProperties);
-		appDeploymentRequest = updateDeploymentProperties(mergedCommandLineArguments, platformName, taskExecutionInformation,
+		appDeploymentRequest = updateDeploymentProperties(commandLineArgs, platformName, taskExecutionInformation,
```

Comment:
I wonder how and why was the `mergedCommandLineArguments` used before. By any chance, can we get this `executeTask` method refactored as this has lots of moving parts and becoming incomprehensible?